### PR TITLE
Floating point and QPTiff metadata support

### PR DESCRIPTION
**Pull request by XanthosXanthopoulos**
This PR adds support for metadata extraction from QPTiff images (both RGB and multichannel) as well as channel min-max computation from floating point images

### CODE REVIEW

File: tiledb/bioimg/helpers.py [hunk 1]
```diff
@@ -169,9 +169,11 @@ def iter_color(attr_type: np.dtype) -> Iterator[Dict[str, int]]:
...
     while True:
         if np.issubdtype(attr_type, np.integer):
-            red = np.random.randint(low=min_val, high=max_val, dtype=attr_type)
-            green = np.random.randint(low=min_val, high=max_val, dtype=attr_type)
-            blue = np.random.randint(low=min_val, high=max_val, dtype=attr_type)
+            red = np.random.randint(low=min_val, high=max_val, dtype=attr_type).item(0)
+            green = np.random.randint(low=min_val, high=max_val, dtype=attr_type).item(
+                0
+            )
+            blue = np.random.randint(low=min_val, high=max_val, dtype=attr_type).item(0)
```
Comment (ktsitsi) [inline tiledb/bioimg/helpers.py:176]: Why does this need to change?

File: tiledb/bioimg/converters/base.py [hunk 2]
```diff
@@ -180,16 +186,70 @@ def from_tiledb(
...
+                                    Axes("".join(axes_metadata.get("originalAxes")))
+                                    .webp_mapper(4)
+                                    .inverse
+                                )
+                        else:
+                            axes_mapper = Axes(
+                                "".join(axes_metadata.get("storedAxes"))
+                            ).mapper(Axes("".join(axes_metadata.get("originalAxes"))))
+
+                    dask_array = dask.array.from_tiledb(member.uri, attribute=attr)
```
Comment (Shelnutt2) [inline tiledb/bioimg/converters/base.py:238]: Why is dask being used here?

Comment (XanthosXanthopoulos) [reply tiledb/bioimg/converters/base.py]: It was a temporary fix to export large images locally and to be removed.

Comment (Shelnutt2) [reply tiledb/bioimg/converters/base.py]: We cannot introduce dask here for that purpose. We need another solution. Dask dependency is not something we should add just for this. If you want chunked writes we can handle that.

cc: @ihnorton

Comment (ktsitsi) [reply tiledb/bioimg/converters/base.py]: We have already a ticket for that, we should plan it and implement it similar to `chunked` ingestion. For now the way we handle the intermediate export with the configurable memory on pods and the bigger memory in local machines will not create problems for most of the files when memory is configured correctly.

**Review by ktsitsi — APPROVED** [review]
LGTM! :tada As discussed let's move the reverted changes to a separate PR.

**Pull request merged**